subprojects/core-api/src/main/java/org/gradle/process/ExecResult.java

```diff
@@ -28,18 +26,18 @@ public interface ExecResult {
     int getExitValue();
 
     /**
-     * Throws an {@link org.gradle.process.internal.ExecException} if the process exited with a non-zero exit value.
+     * Throws an {@link ProcessExecutionException} if the process exited with a non-zero exit value.
```

Comment:
💅 Should be "Throws a" not "Throws an" now.

Reply:
Good catch